pkgs/network/src/ethlibp2p.zig

```diff
@@ -159,12 +159,7 @@ pub const EthLibp2p = struct {
         const local_private_key = try self.allocator.dupeZ(u8, self.params.local_private_key);
 
         var topics_list: std.ArrayListUnmanaged([]const u8) = .empty;
-        defer {
-            for (topics_list.items) |topic_str| {
```

Comment:
Why remove this？I thought it need to free the topic str in the list. And even though it not need, it is better to create specific PR for it.